src/pages/home/components/howToContact.jsx

```diff
@@ -19,4 +19,12 @@ const Wrapper = styled.div`
   margin-left: auto;
 `;
 
+const Subheader = styled.h1`
+  font-size: 56px;
+`;
+
+const Subtext = styled.p`
+  font-size: 24px;
+`;
```

Comment:
Should this be externalized so it can be used elsewhere for consistency rather than having each file defining its own styles independently?

Reply:
Yes it should be, and will be fixed later. I created a new issue for this, #50, because there is quite a bit I need to fix in regards to styling, however I wanna put that off until we do a thorough review of styling on the site